gtsfm/frontend/matcher/twoway_matcher.py

```diff
@@ -86,3 +85,55 @@ def match(
         match_indices[:, 1] = valid_idx_i2[match_indices[:, 1]]
 
         return match_indices
+
+    def __init_opencv_matcher(self) -> cv.DescriptorMatcher:
+        """Initialize the OpenCV matcher.
+
+        Returns:
+            Matcher object.
+        """
+        if self._distance_type is MatchingDistanceType.EUCLIDEAN:
+            distance_metric = cv.NORM_L2
+        elif self._distance_type is MatchingDistanceType.HAMMING:
+            distance_metric = cv.NORM_HAMMING
```

Comment:
Do we have any binary descriptors in GTSFM?

Reply:
We don't right now. But I prefer to just keep it and add an issue to track removal if we want it removed.